### PR TITLE
Release notes for Tizen studio 3.1 release

### PR DESCRIPTION
**Pull request by vivek-ellur**
Signed-off-by: vivek ellur <vivek.ellur@samsung.com>

### Change Description ###

Add Tizen Studio 3.1 Release Notes .

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/525

**Comment by mijin85cho** [comment]
Hello, @annie-samsung.
As written in tizen-studio/release-notes/3-1-release-notes.md updated by this PR,
Tizen Studio 3.1 is planed to be released by the mid of this week.

So, please give this PR high priority that we can publish this page by this Wednesday

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+-   Release Date: Jan 15, 2019
+
+
+## IDE and Tools
+
+
+### New Features
+-   OpenJDK
+	-  Added support for OpenJDK 10
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: It is OpenJDK 10

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### Fixed Bugs
+-   Tizen IDE
+	-   Changing the profile and platform version now updates the rootstrap in project configuration.
+	-   Fixed the error message when adding incorrect privilege file to a native application.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	-   Fixed issues with Log filtering in log tab
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+-   Certificate manager
+	-   Fixed the failure in generating a new certificate profile after 01 January 2019
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: The existing sentence is correct

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+	-   Network proxy setting change is applied for all network accesses
+	-   Made unnecessary input fields optional in UI when creating IoT certificates
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+	-  Added support for OpenJDK 10
+
+### Fixed Bugs
+-   Tizen IDE
+	-   Changing the profile and platform version now updates the rootstrap in project configuration.
+	-   Fixed the error message when adding incorrect privilege file to a native application.
+	-   Fixed issues with Log filtering in log tab
+-   Certificate manager
+	-   Fixed the failure in generating a new certificate profile after 01 January 2019
+	-   Network proxy setting change is applied for all network accesses
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+	-   Made unnecessary input fields optional in UI when creating IoT certificates
+-   Device Manager
+	-   Fixed issues with Log filtering in log tab
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+-   TizenRT
+	-   IoT certificates are copied to the project while building instead of flashing
+	-   RT IDE isntallation prompts user to install libwebkitgtk if not installed to prevent "browser path not set" error pop up
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+	-   Fixed the error message when adding incorrect privilege file to a native application.
+	-   Fixed issues with Log filtering in log tab
+-   Certificate manager
+	-   Fixed the failure in generating a new certificate profile after 01 January 2019
+	-   Network proxy setting change is applied for all network accesses
+	-   Made unnecessary input fields optional in UI when creating IoT certificates
+-   Device Manager
+	-   Fixed issues with Log filtering in log tab
+-   TizenRT
+	-   IoT certificates are copied to the project while building instead of flashing
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/toc_application.md [hunk 10]
```diff
@@ -1215,7 +1215,7 @@
 ##### [Device/Resource Model Manager](/application/tizen-studio/rt-ide/smartths-ext/model-manager.md)
 
 ### Release Notes
-
+#### [3.1 (Jan 15, 2019)](/application/tizen-studio/release-notes/3-1-release-notes.md)
```
Comment (mijin85cho) [inline docs/application/toc_application.md]: This toc_application.md file is to backup the toc_all.md file version of #512.
You don't need to update toc_application.md file.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
+# Tizen Studio 3.1 Release Notes
+
+-   Release Date: Jan 15, 2019
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Please update this line with actual date

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Reni-90) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: OK.
Please add a period (.) at the end of the sentence.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md (hunk 11 above)
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Hello,
I talked with Mr. Chang-seok Oh. he told that document team will update the date when tizen studio is released

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+-   Device Manager
+	-   Fixed the issues with log filtering in log tab.
+-   TizenRT
+	-   Fixed the issue in RT IDE . Now, the IoT certificates are copied to the project while building instead of flashing.
+	-   RT IDE installation prompts user to install libwebkitgt to avoid "browser path not set" error message.
+	-   Fixed the issues that occurs while debugging Tizen RT application. 
+
+### Known Issues
+-	Tizen Studio
+	-	In case of Mac, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+	-   On Windows&reg;, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
+	-   On Windows&reg;, if an error with message "failed to allocate memory" occurs while executing the emulator, try the following:
+	    -   Close some other programs and try to launch the emulator again.
+	    -   If the RAM size is set to 768 or 1024 MB for the VM in the Emulator Manager, change it to 512 MB.
+	    -   Increase the user area of the virtual memory in the system to 3 GB by entering the **bcdedit /setincreaseuserva 3072** command on the console with administrator rights (only in Windows&reg; 7), and reboot.
+	-   If you use a MacBook Pro which has both Intel HD and NVIDIA&reg; GPUs, the emulator can unexpectedly terminate when you execute the emulator with OpenGL ES version 1.1 or 2.0. Verify the emulator configuration in the Emulator Manager and on the general tab in the emulator configuration window, set OpenGL ES version to version 2.0 or to version 3.0.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Done

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/525

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Reni-90) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: @mijin85cho 
I had the discussion with Annie. We will update the changes with "have been pp" for both Tizen Platform and Studio release notes.
Can we fix the release notes post 3.1 release. Please guide me.

**Comment by mijin80lee** [comment]
TD Sync Completed.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: "Can we fix the release notes post 3.1 release. Please guide me."
-> I'm not sure with this sentence. Are you meaning the blog post?
The blog post doesn't need to follow "have been pp" rule.  So you don't need to fix to follow "have been pp".
Please check with Annie.

[the remark this answers is not in the document]
Comment (Reni-90) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: I am sorry. My intention was to ask whether we can update the release notes after closing the current PR. As this is an urgent PR for 3.1 release. Nothing regarding the blog post.
We would like to fix all the release notes (both Platform and Studio) in same format.

**Pull request merged**